src/main/java/com/github/dockerjava/api/command/CreateContainerCmd.java

```diff
@@ -298,6 +301,10 @@
      */
     CreateContainerCmd withDnsSearch(List<String> dnsSearch);
 
+    CreateContainerCmd withDnsOptions(String... dnsOptions);
```

Comment:
Please keep only list, this will shift code coverage to single method. I don't think that it too complex to pass list instead array.